course-material/intro-to-sql/combining-data-from-multiple-tables.md

```diff
@@ -24,16 +32,226 @@ This code installs JupySQL, DuckDB, and Pandas in your environment. We will be u
 ```
 
 ## Load the data
-We extract the financial data by retrieving it from it's URL download link. The link may be a zip file (which it is in this case), so we extract the zip file and conver the .asc files to .csv files. Finally, we save converted data into a folder.
+
+```{important}
+<b>Note:</b> If you are following these lessons locally and <b>not</b> on Google Colab, then there is no need to load the data again. 
+```
+
+This section was covered in detail in the previous tutorial: [Joining Data in SQL](https://ploomber-sql.readthedocs.io/en/latest/intro-to-sql/joining-data-in-sql.html#load-the-data). We will be using the same data in this tutorial as well.
 
 ```{code-cell} ipython3
 import banking_data_script
-
 # ZIP file download link
 link = "http://sorry.vse.cz/~berka/challenge/pkdd1999/data_berka.zip"
 # Naming our folder that will hold our .csv files
 output = "expanded_data"
 banking_data_script.extract_asc_to_csv(link, output)
 ```
 
+If you ran the above cell, you should have a folder `expanded_data` in your current directory that contains the `.csv` files we will be using. However, in this tutorial, we shall focus on four out of the eight `.csv` files.
+
+## ER Diagram
+
+In the previous tutorial, we provided an [Entity-Relationship Diagram (ERD)](https://ploomber-sql.readthedocs.io/en/latest/intro-to-sql/joining-data-in-sql.html#datasets) to help us understand the relationship between the `account` and `district` tables. In this section, we shall be augment that diagram to understand the relationship between multiple tables.
+
```

Comment:
"We shall augment" -> "we shall be augment"

Reply:
Suggestion, replace instances of "shall" with "will" 

Reads a bit more approachable, less formal